spec/generators/dashboard_generator_spec.rb

```diff
@@ -36,12 +36,44 @@ class Foo < ActiveRecord::Base
       end
 
       it "includes user-defined database columns" do
```

Comment:
Updated this spec to test that the correct classes & constants actually get defined, instead of just looking for the correct strings in the generated file.